defaults/main.yml

```diff
@@ -80,4 +80,4 @@ gitlab_letsencrypt_contact_emails: ["gitlab@example.com"]
 gitlab_letsencrypt_auto_renew_hour: 1
 gitlab_letsencrypt_auto_renew_minute: 30
 gitlab_letsencrypt_auto_renew_day_of_month: "*/7"
-gitlab_letsencrypt_auto_renew: true
+gitlab_letsencrypt_auto_renew: "true"
```

Comment:
Thank a lot for getting this to work :+1: 

For clarity and simplicity instead of inputting a bool as a string inside vars, maybe it would be better to do this in the template: 
`{{ gitlab_letsencrypt_auto_renew|lower }}` and all other boolean variables that are wrongly appended with capitals that are not understood by gitlab

bu nonetheless if it works don't touch it :P

Reply:
I like this idea more (`{{ gitlab_letsencrypt_auto_renew|lower }}` and leave the vars in the defaults the way they are. It protects against people unintentionally switching between string / boolean (if they set `false` or `"false"` it will still work).

Reply:
I've accepted that pull request (thanks @giorgioma)!

@geerlingguy any other changes you would like to see?

Reply:
Or wait, did I misunderstand? Do you prefer to have the changes in templates/gitlab.rb.j2 but revert the changes in defaults/main.yml (the quoting)?

Reply:
I did change `defaults/main.yml` to match the use of booleans variables ;)